app/models/ability.rb

```diff
@@ -0,0 +1,13 @@
+# frozen_string_literal: true
+
+class Ability
+  include CanCan::Ability
+
+  # User who made the request
+  def initialize(user)
+    # defined what can any user do with and article, 
+    # Had to be specified the relation between user and article (owner : user.id)
+    can :manage, Article, owner: user
```

Comment:
@sansuaza how CanCan works?

Reply:
When the gem cancancan is imported into the project, the ability class is generated with the cancan:ability command. In this class the access rules for the logged user are indicated.
In this case, only if the user is the owner of the Article, is able to manage it